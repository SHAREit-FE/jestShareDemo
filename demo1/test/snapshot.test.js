//快照：https://jestjs.io/docs/zh-Hans/snapshot-testing
import config from "../demo/snapshot";
test('snapshot 功能',()=>{
    expect(config()).toMatchSnapshot();
})