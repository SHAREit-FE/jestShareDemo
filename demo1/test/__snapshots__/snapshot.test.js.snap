// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 功能 1`] = `
Object {
  "age": 18,
  "name": "mandy2",
}
`;
